packages/core/src/App/Containers/RealAccountSignup/add-crypto-currency.jsx

```diff
@@ -115,27 +115,30 @@ const AddCryptoCurrency = ({
                     )}
                     {!should_show_fiat_only &&
                         (available_crypto_currencies.length !== 0 ? (
-                            <CurrencyRadioButtonGroup
-                                id='crypto_currency'
-                                className='currency-selector__radio-group currency-selector__radio-group--with-margin'
-                                label={localize('Cryptocurrencies')}
-                                value={values.currency}
-                                error={errors.currency}
-                                touched={touched.currency}
-                                is_title_enabled={canAddFiat()}
-                                item_count={getReorderedCryptoCurrencies().length}
-                            >
-                                {getReorderedCryptoCurrencies().map(currency => (
-                                    <Field
-                                        key={currency.value}
-                                        component={CurrencyRadioButton}
-                                        name='currency'
-                                        id={currency.value}
-                                        label={currency.name}
-                                        selected={canAddCrypto(currency)}
-                                    />
-                                ))}
-                            </CurrencyRadioButtonGroup>
+                            <Text as='div' align='center' size='s' weight='bold' color='prominent'>
+                                <CurrencyRadioButtonGroup
+                                    id='crypto_currency'
+                                    className='currency-selector__radio-group currency-selector__radio-group--with-margin'
+                                    label={localize('Cryptocurrencies')}
```

Comment:
i.e., If it's only for the label, this could be: 

```suggestion
                                    label={
                                        <Text as='div' align='center' size='s' weight='bold' color='prominent'>
                                            localize('Cryptocurrencies')
                                        </ Text>
                                    }
```

Reply:
Noted, thanks